eve/workers/end2end/scripts/run-e2e-test.sh

```diff
@@ -108,4 +108,6 @@ elif [ "$STAGE" = "backbeat" ]; then
    run_e2e_test '' 'cd node_tests && npm run test_all_extensions && cd .. && python3 cleans3c.py'
 elif [ "$STAGE" = "iam-policies" ]; then
    run_e2e_test '' 'cd node_tests && npm run lint && npm run test_iam_policies'
+elif [ "$STAGE" = "object-api" ]; then
+   run_e2e_test '' 'cd node_tests && npm run lint && npm run test_object_api'
```

Comment:
seems strange that `lint` is called twice, both in `iam-policies` and `object-api` : should it be refactored in a separate CI step?

Reply:
Thinking about it, we might want to run the linter for all of stages (except `debug`), the linter was just recently added to node tests so it wasn't added in older stages

Reply:
Yes, we added a linter recently as we added new tests in Zenko CI. We found that the old tests were not using any linter, making the test code hard to read. The linter was then restricted to iam-policies tests for now. Killian did another PR to update all the `node_tests` so there is no reason to still use this here. I would suggest adding a new step when building to run the `npm run lint` once for all tests.